tests/ztimer_underflow/Makefile

```diff
@@ -0,0 +1,13 @@
+include ../Makefile.tests_common
+
+USEMODULE += ztimer
+
+# Which clock shall be tested?
+# -> ZTIMER_USEC
```

Comment:
can we make this a bit easier to use from CLI?
e.g., ```
TEST_ZTIMER_CLOCK ?= ZTIMER_USEC
CFLAGS += -DTEST_ZTIMER_CLOCK=$(TEST_ZTIMER_CLOCK)

... some logic to add usemodule depending on $(TEST_ZTIMER_CLOCK)
```

That way one can do `make TEST_ZTIMER_CLOCK=ZTIMER_MSEC ...`, without editing the file.

Reply:
Good idea.
Do you have an idea how to handle the `USEMODULE += ztimer_periph_rtt`? It cannot be required if the board has no `periph_rtt`.

Reply:
Done.